2018---The REAL First Power Up/src/org/usfirst/frc/team2586/robot/Robot.java

```diff
@@ -467,7 +471,7 @@ private boolean autoDrive(double distance) {
 
 		// Determine if the robot made it to the target
 		// and then wait a bit so that it can correct any overshoot.
-		if (e_lin > 0.5 || e_rot > 2.0)
+		if (e_lin > 40 || e_rot > 2.0)
```

Comment:
I think this is because of your scale factor being wrong. As written the robot would need to be within +-40 inches to be @ target. Seems a bit too broad. 

Maybe 2 inches would be better, I wouldn't go much more than that.

If the robot doesn't have enough power to make it to the target, increase kP. If it overshoots, reduce it.

Reply:
I'll change that, apparently this version didn't have my minimum output changes(which apparently could have been completely avoided by just screwing with the kP value)

Reply:
Honestly, since this is our last day to work I don't think I'll have time to fix it and test/debug it. What I have is working really consistently(again on branch #2) and have other stuff to do(update firmware for victors, etc)